admin/commands/inventory/inventory.go

```diff
@@ -23,9 +23,9 @@ import (
 
 // InventoryCommand is used by Kong for CLI flags and commands.
 type InventoryCommand struct {
-	List   ListCommand   `cmd:"" hidden:"" help:"List inventory commands"`
```

Comment:
Was the `hidden` tag meant to prevent the command/subcommand from being shown in the help text?

Reply:
Yes, it was meant not to show up, but such an approach turned out to be counter productive and was reported as a bug. It's quite enough to hide the top-level `pmm-admin inventory` command, which is in effect hidden right now.